spec/controllers/instruments_controller_spec.rb

```diff
@@ -535,11 +531,11 @@ def assert_successful_update
           do_request
         end
 
-        it "should show the primary instrument's admin reservation" do
+        it "should show the primary instrument's reservations" do
```

Comment:
Are these two tests (at 534 and 538) redundant given what the one at line 542 does?